docs/docs/draft/documentation/README.md

```diff
@@ -308,6 +308,42 @@ kuma.io/sidecar-injection: enabled
 
 On Kubernetes the [`Dataplane`](#dataplane-entity) entity is also automatically created for you, and because transparent proxying is being used to communicate between the service and the sidecar proxy, no code changes are required in your applications.
 
+### Gateway
+
+The `Dataplane` can operate in Gateway mode. This way you can integrate Kuma with existing API Gateways like [Kong](https://github.com/Kong/kong).
+In Gateway mode, ingress traffic won't be intercepted and Envoy won't expose inbound listeners.
```

Comment:
> In Gateway mode, ingress traffic won't be intercepted and Envoy won't expose inbound listeners.

I would suggest to explain it in less technical terms.
This sentence only mentions what will not happen. But what will happen ? How does the traffic flow look in this case ?

Reply:
Expanded the description. Is it better now?